pipeline-scripts/create-database.sql

```diff
@@ -820,20 +820,29 @@ CREATE PROCEDURE CreateDatabase ()
             UNIQUE chain_type_name_uq (Name)
         ) ENGINE = INNODB;
 
-        CREATE TABLE IF NOT EXISTS token_chain(
+        CREATE TABLE IF NOT EXISTS token_wrapped(
             Id                  BIGINT UNSIGNED AUTO_INCREMENT,
             TokenId             BIGINT UNSIGNED NOT NULL,
+            Owner               VARCHAR(50) NOT NULL,
             NativeChainTypeId   SMALLINT UNSIGNED NOT NULL,
             NativeAddress       VARCHAR(100) NULL,
+            CreatedBlock        BIGINT UNSIGNED NOT NULL,
+            ModifiedBlock       BIGINT UNSIGNED NOT NULL,
```

Comment:
Should we add the `Trusted` bit flag?

Reply:
Potentially, or we create a `trusted_custodian` table? There might be more than 1 and would save us having to update the database if it changes

Reply:
Now I feel like we're pulling a thread that will keep going 😅 I guess we should skip for now and reevaluate once we get this first step done. 

We check at runtime within the token assemblers to compare the configurations against the owner to return a `Trused` flag if they match.

Reply:
I'll do what you suggested for now and simply use the config to create a trusted flag. You're right about the thread continuing, as something that occurred to me is that if we want to future-proof it (avoiding breaking changes in the API), we should consider additional wrapping methods alongside Interflux. In which case we might want to also return out to the API `bridgeType: Interflux` and we'd need different 'trusted' addresses for each bridge type.